### ISSUE TITLE
Component value is not cleared when the bound value is undefined

### ISSUE THREAD

**Issue by len-ro** [aurelia-ui-toolkits/aurelia-kendoui-bridge#623]
Can be tested here: https://gist.run/?id=e7b079763ddf2c43587d88171d5ff82b, the new button does not clear the datePicker or the NumericTextBox. Setting the bound value to null clears the datePicker however but it is not always applicable.

**Comment by len-ro** [comment]
At a first glance adding:

```
if(typeof newValue == "undefined"){
		widget[binding.functionName](null);
	    }
```
in 

```
handlePropertyChanged(widget, property, newValue, oldValue) {
    let binding = this.bindingsToKendo.find(i => i.propertyName === property);
    if (binding) {
      widget[binding.functionName](newValue);
    }
  }
``` widget-base fixes the pb.

**Comment by JeroenVinke** [comment]
released in 1.0.0-beta.1.0.14
